source/process/Recruitment

```diff
@@ -0,0 +1,137 @@
+Our mission at Mattermost is to be the primary way people communicate within Enterprise and applications. 
+ Our goals are to become the leading enterprise-grade message solution, we’ll achieve this by: 
```

Comment:
Flow for this sentence needs a bit of work, should it be "Our goal is to become" and "messaging solution"? 

I think the intro to the list could be improved as well, ideally it should read like a sentence: 
"We’ll achieve this by #1 private cloud solution by deployments (12-18 months)" doesn't make a lot of sense